src/dkg.ts

```diff
@@ -85,15 +90,16 @@ export class DkgRitual {
 }
 
 export class DkgClient {
-  constructor(private readonly provider: ethers.providers.Web3Provider) {}
-
   // TODO: Update API: Replace with getExistingRitual and support ritualId in Strategy
-  public async initializeRitual(ritualParams: {
-    shares: number;
-    threshold: number;
-  }): Promise<DkgRitual> {
+  public static async initializeRitual(
+    web3Provider: ethers.providers.Web3Provider,
+    ritualParams: {
+      shares: number;
+      threshold: number;
+    }
+  ): Promise<DkgRitual> {
     const ritualId = 2;
```

Comment:
I assume all of this is only for testing. Is this something that can be moved to tests / mocked in tests instead of including it in actual code?

Reply:
My assumption was that for the time being we're going to operate on a set of premade rituals, and right now we only have one

Reply:
On second thought I think it's reasonable to expose `ritualId` as a parameter at least within the SDK

Reply:
Implemented in #233